lock-conjure-api/src/main/conjure/lock-api.yml

```diff
@@ -26,7 +26,9 @@ types:
           expirationDateMs: Long
       ConjureLockV1Request:
         fields:
-          lockClient: string
+          lockClient:
+            type: string
+            safety: safe
           lockRequest: LockRequest
```

Comment:
Has it been made safe by the way it is used?

Reply:
This is a user-provided string. Admittedly I've been relying on the existing LockService API, which at least since 2017 has this tagged explicitly as `@Safe`.